org.eclipse.xtext.ui.tests/tests/org/eclipse/xtext/ui/tests/xmleditor/XmlParsingTest.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2019, 2021 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.ui.tests.xmleditor;
+
+import org.eclipse.xtext.testing.InjectWith;
+import org.eclipse.xtext.testing.XtextRunner;
+import org.eclipse.xtext.testing.util.ParseHelper;
+import org.eclipse.xtext.testing.validation.ValidationTestHelper;
+import org.eclipse.xtext.ui.tests.xmleditor.simplexml.XmlDocument;
+import org.eclipse.xtext.ui.tests.xmleditor.tests.XmlInjectorProvider;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.google.inject.Inject;
+
+@RunWith(XtextRunner.class)
+@InjectWith(XmlInjectorProvider.class)
+public class XmlParsingTest {
+	@Inject
+	private ParseHelper<XmlDocument> parseHelper;
+
+	@Inject
+	private ValidationTestHelper validationTestHelper;
+
+	@Test
+	public void test001() throws Exception {
+		XmlDocument model = parseHelper.parse("");
+		validationTestHelper.assertNoErrors(model);
```

Comment:
Create a helper method "assertNoErrors" that does the parsing and the delegation to simplify the code to "assertNoErrors("")".

Reply:
dont want to start a big refactoring